### ISSUE TITLE
[BUG] Adding colons to image alt text causes TTW book builds to fail

### ISSUE THREAD

**Issue by f-rower** [the-turing-way/the-turing-way#3736]
### Description

Whenever alt text in an image includes colons, the TTW checks fail, particularly the following ones:
- CI / Build (ubuntu-latest, 3.8) (pull_request) 
- CI / Build (macos-latest, 3.8) (pull_request) 
- CI / Build (windows-latest, 3.8) (pull_request)

Removing colons fixes the problem

Is there any way we can allow colons into the alt-text?

### How to reproduce

Include an image into a file

This alt-text WOULD work

```{figure} ../../figures/turing-roche-partnership.jpg
---
height: 500px
name: turing-roche-partnership
alt: Scriberia illustration showing the journey of the Turing-Roche partnership over 5 years.
---
The Turing-Roche Strategic Partnership
_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. Original version on Zenodo. [http://doi.org/10.5281/zenodo.8169292](http://doi.org/10.5281/zenodo.8169292) 
```

This alt-text WOULD NOT work

```{figure} ../../figures/turing-roche-partnership.jpg
---
height: 500px
name: turing-roche-partnership
alt: Scriberia illustration showing the journey of: the Turing-Roche partnership over 5 years.
---
The Turing-Roche Strategic Partnership
_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. Original version on Zenodo. [http://doi.org/10.5281/zenodo.8169292](http://doi.org/10.5281/zenodo.8169292) 
```

## Steps
Some possible steps to take whilst this bug is resolved could involve the following:
- [ ] Updating the avoiding certain characters in the [Alt-text](https://the-turing-way.netlify.app/community-handbook/accessibility/alt-text#ch-accessibility-alttext) chapte
- [ ] Guidance on [figures](https://the-turing-way.netlify.app/community-handbook/style/style-figures)

**Comment by JimMadge** [comment]
Makes sense as that is a yaml block.

Can you escape the colon (`\:`)?

**Comment by RichardJActon** [comment]
Perhaps in style guide (https://book.the-turing-way.org/community-handbook/style/style-figures#caption) we should recommend using the 'folding block scalar' yaml syntax by default here, as I've seen at least three people encounter similar issues?

```
alt: >
  alternative text: which can include unescaped special characters except \> and \|.
```

**Comment by JimMadge** [comment]
@RichardJActon That is a good idea. Unless there is an update I think this is not a bug.

**Comment by aleesteele** [comment]
This sounds great @RichardJActon - updating our Alt text section would also be great: https://book.the-turing-way.org/community-handbook/accessibility/alt-text
